Cargo.toml

```diff
@@ -60,10 +60,10 @@ async-trait = "0.1"
 base64 = "0.21"
 bytes = "1"
 chrono = "0.4.24"
-dirs = "5"
 form_urlencoded = "1"
 hex = "0.4"
 hmac = "0.12"
+home = "0.5"
```

Comment:
`home` depends on deprecated API: `std::env::home_dir()` which is not a good choice. What do you think?

Reply:
It is only deprecated for Windows & they only use it for non-windows. They've a separate logic written just for Windows.